docs/v0.13.0-release-notes.md

```diff
@@ -0,0 +1,149 @@
+
+Fixes #334. Thanks @stevesloka.
+
+### Sample `deployment/` YAML examples moved to `examples/`
+
+Since our 0.1 release Contour has always included in the repository sample YAML for various configurations.
+These were always intended to be _examples_, and this is how the Contour team always perceived them.
+However we did a bad job of communicating this to our userbase, which we are now trying to correct.
+
+In operation nothing has changed with the sample YAML other than it has moved from `deployment/` to `examples/` to make clarify that these are in fact simply examples.
+
+Fixes #1118. Many thanks to @rochacon.
+
+### `--envoy-external-http-port` and `--envoy-external-https-port` flags have been deprecated
+
+Due to a long standing limitation in Envoy, if Contour was deployed on ports other than the tradition 80 (HTTP), and 443 (HTTPS), operators were required to pass to Envoy, via `--envoy-external-http-port` and `--envoy-external-https-port`, the non standard ports that were in use. This was annoying in practice, and restricted the use of local development tools like Minikube and Kind.
+
+Contour 0.13.0 introduces a workaround for envoyproxy/envoy#1269, that removes the need to inform Envoy of external ports that will be forwarded too it. In turn this should make it easier to deploy Contour inside Kind or Minikube clusters.
+
+As they are no longer needed, the `--envoy-external-http-port` and `--envoy-external-https-port` flags now generate a warning if used and will be removed completely in 0.14.0.
+
+Fixes #210. Thanks @youngnick.
+
+### `force-ssl-redirect` now takes precidence over the `ingress.allow-http` annotation
+
+The behaviour when the `kubernetes.io/ingress.allow-http` and `ingress.kubernetes.io/force-ssl-redirect` were both specified was somewhat surprising. `ingress.allow-http: false` meant that no routes were not registered for port 80, even if `force-ssl-redirect: true` was set  leading to a 404 where a 3xx upgrade to `https` was expected.
+
+Contour 0.13.0 now prioritizes `force-ssl-redirect`. If this annotation is specified and set to `true`, Contour will always register a port 80 route for the ingress, even if `ingress.allow-http: false`, so that the forced upgrade can take effect.
+
+Fixes #1023 with many thanks to @ceralena.
+
+### `Maglev` and `RingHash` load balancer strategies no longer supported.
+
+`RingHash` and `Maglev` are two balancing / affinity strategies offered by IngressRoute. However due to a lack of understanding of how they worked when they were added in Contour 0.6, neither strategy was properly configured and would only result in random behaviour. 
+Without the ability to configure the hash key, which is usually some for of session af session or cookie, these strategies are not useful and cannot be used correctly.
```

Comment:
"some for of session af session or cookie" -> "some form of session or cookie"?

Reply:
wow, two big mistakes in the same paragraph. Fixed, thankyou.